app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+
+#### Bursaries for physics and languages
+
+You do not need to apply for a bursary – it will automatically be awarded to you through your training provider if you are eligible.
+
+You may get a:
+
+* physics bursary of £27,000
+* languages bursary of £25,000
+
+To be eligible, you’ll need a place on a course training to teach languages or physics with an accredited teacher training provider in England, in the academic year 2023 to 2024.
+
+You’ll also need at least one of the following:
+
+* a bachelor’s degree class 2:2 (honours) or higher
+* a master’s degree
+* a PhD
+
+Each bursary scheme will have their own individual terms and conditions – contact your chosen teacher training provider for more information.
+
+The languages bursary covers all languages offered by accredited teacher training providers. If a course combines 2 subjects (for example, physics plus mathematics, or modern foreign languages plus English), at least 50% of the course must be made up of the bursary subject.
+
+
+#### Scholarships for physics, French, German and Spanish
+
+To get a scholarship, you’ll need to be a high-quality student and pass a tough selection process, run by the professional body for your subject.
+
+If you’re successful, your teacher training provider will pay the scholarship money directly into your bank account over a period of 10 months.
+
+You may get a:
+
+* physics scholarship of £29,000
+* French, German or Spanish scholarship of £27,000
+
+To apply for a scholarship, you’ll need a place on a course training to teach French, German, Spanish or physics with an accredited teacher training provider in England, in the academic year 2023 to 2024.
+
+You’ll also need at least one of the following:
+
+* a bachelor’s degree class 2:1 (honours) or higher
+* a master’s degree
+* a PhD
+
+In exceptional circumstances, you may be offered a scholarship with a bachelor’s degree class 2:2, if you have significant relevant experience.
+You can check if you’re eligible for a scholarship and apply by visiting:
+
+* the [Institute of Physics](https://www.iop.org/about/support-grants/iop-teacher-training-scholarships#gref) (for a physics scholarship) 
+* the [British Council](https://www.britishcouncil.org/education/he-science/opportunities/ltts) (for a French, German or Spanish scholarship)
+
+[Find teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in languages and physics. 
+
+## Eligibility for other types of financial support
+
+Depending on your immigration status, you may be able to get other kinds of financial help from the UK government. 
+
+You are more likely to get financial support if you have some type of settled status in the UK – for example, through the EU Settlement Scheme or indefinite leave to remain. Without settled status, most non-UK candidates will not be eligible for financial support.
+
+Financial support is only available for courses where you pay a fee to study teacher training. 
```

Comment:
Is this true? I thought earlier on, you said candidates will be able to apply for the IRP if they're on a salaried course in autumn?

Reply:
Reword this to say tuition fees and loans